.buildkite/pipeline.yml

```diff
@@ -92,6 +103,14 @@ steps:
           slurm_mem_per_cpu: 8G
           slurm_gpus: 1
 
+      - label: "Global Bucket on GPU"
+        key: "global_bucket_gpu"
+        command: "julia --color=yes --project=experiments experiments/standalone/Bucket/global_bucket.jl"
+        agents:
+          slurm_mem_per_cpu: 8G
```

Comment:
should we specify `slurm_mem_per_cpu` for GPU runs? I see in ClimaAtmos they just specify `slurm_mem`, I'm not sure what's best or if they're equivalent https://github.com/CliMA/ClimaAtmos.jl/blob/main/.buildkite/pipeline.yml#L691-L692

Reply:
Ill update this